index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable import/no-named-as-default-member */
+/* eslint-disable no-console */
+// eslint-disable-next-line import/extensions
+/* eslint-disable import/extensions */
+/* eslint-disable no-console */
+/* eslint-disable no-console */
+// eslint-disable-next-line import/extensions
```

Comment:
don't ignore any eslint rule
and all source file must be inside the server folder so this shouldn't be here

Reply:
All source files are now in src folder